openslides/core/static/index.html

```diff
@@ -0,0 +1,3 @@
+<h1>Hello user!</h1>
+
+Use <a href="http://localhost:4200">This url</a> to get to the client. For further information see the DEVELOPMENT.rst
```

Comment:
Why not add a Django redirect view?

Reply:
When you call yarn build, this index.html is overwritten and the actual client is returned.

Of cause, you could add some JS in this html file, that makes the redirect.

Reply:
This may not be the right url. We better give hints how to set up the server and client rigt, because if one gets to the page, he did something wrong